content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,69 +6,86 @@ aliases:
 
-* New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.
+## What is the migration timeline?
 
-* Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).
+<strong>March 5, 2022</strong> - The new Events Explorer and analytics were made available. We began to migrate customer dashboards and monitors not managed using our API.
 
-This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).
+<strong>April 30, 2022</strong> - The event stream will be retired in favor of the Event Explorer.
 
-## Migration process
+<strong>May 15, 2022</strong> - Starting on this date, while Datadog will continue to evaluate event monitors that have not been migrated, they will no longer be editable. New event monitors must use the new syntax.
 
-### Datadog migrates monitors and dashboards configured through the UI
-<div class="alert alert-info">
-You do not need to take any action for dashboards or monitors created through Datadog.
-</div>
+<strong>June 30, 2022</strong> - On this date, Datadog will stop evaluating event monitors that have not been migrated.
 
-The changes to Event query syntax affect two Datadog features: monitors and dashboard widgets.
+## What action do I need to take?
 
-Datadog migrates monitors that query Events by doing the following:
-- For each legacy Event monitor, creates a duplicate that uses the new query syntax. Enables the new monitor.
-- Adds the prefix "legacy" to the legacy Event monitor, and mutes it.
-- Updates all SLOs and monitor downtimes that reference the legacy Event monitor to point to the new monitor.
+If you do <strong>not</strong> manage your dashboard or monitors using external API-based tools (such as Terraform or scripts), <strong>then no action is required on your end</strong>. Datadog will migrate your dashboards and monitors before April 30, 2022. We will leave your old monitors in place but they will be muted and Datadog will stop evaluating them on June 30, 2022.
 
-When viewing monitors, you can filter which of the two monitor types you would like to see using the **Event Migration** facet. This facet has available values `Event Monitors V1` and `Event Monitors V2`.
+<strong>If you use Terraform or other API-based scripts</strong> to manage all or some of your <strong>dashboards</strong>, Datadog will migrate queries in your event widgets and overlays to the new syntax, but you will need to update your scripts to keep them in sync before June 30, 2022.
```

Comment:
Can we include a link to the migration script in this section and the next one to give them more details about how they can update their scripts?